app/views/movies/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<h2><%= @movie.title %></h2>
+<p><b>Studio: </b><%= @movie.studio.name %></p>
+<p><b>Creation Year: </b><%= @movie.creation_year %></p>
+<p><b>Genre: </b><%= @movie.genre %></p>
+<p><b>Actors: </b><br><% @movie.actors.sort_asc_age.each do |a|%>
+                       ID: <%= a.id %>  |  <%= a.name%> | Age: <%= a.age%><br>
+                       <% end%>
+
+<p><b>Average Age of Actors:</b> <%= @movie.average_age %> years old 
+
+<h3>Add an Existing Actor to Movie by ID</h3>
+  <%= form_with url: "/movies/#{@movie.id}", method: :get, data: { turbo: false } do |form| %>
```

Comment:
The way you have this set up with the search form + button combo, I agree that this would be a `:get`. If we combined it all into one form, then the method should be `:post`. This is ok for now, but be careful about this kind of approach; I wouldn't want you to lose time in the final or retake is all.